### PR TITLE
Fix dbBottomEntries bug

### PR DESCRIPTION
**Pull request by Satty9361**
There was a bug since last update. dbBottomEntries should still sort values in ascending order.

### CODE REVIEW

File: customcommands/tmplextensions.go [hunk 1]
```diff
@@ -532,7 +532,7 @@ func tmplDBCount(ctx *templates.Context) interface{} {
 func tmplDBTopEntries(ctx *templates.Context, bottom bool) interface{} {
 	orderBy := "value_num DESC, id DESC"
 	if bottom {
-		orderBy = "value_num DESC, id ASC"
+		orderBy = "value_num ASC, id ASC"
```
Comment (jonas747) [inline customcommands/tmplextensions.go]: Maybe also invert id ASC so that its the opposit of topentries?

**Comment by Satty9361** [comment]
I am probably missing something but the order is already opposite. It is id DESC in topentries and id ASC in bottomentries. Do you want me to make both same ?

**Comment by jonas747** [comment]
Ah nvm then, thats's correct

**Comment by Satty9361** [comment]
Done

**Comment by jonas747** [comment]
sorry i already did it on my end since i wanted to get it out fast

**Pull request closed without merging**